apps/teams-test-app/src/pages/TestApp.tsx

```diff
@@ -71,6 +71,7 @@ export const TestApp: React.FC = () => {
 
   return (
     <>
+      <button onClick={() => window.location.reload()}>Reload This App</button>
```

Comment:
If you want to call this from tests it should have an id so it can be retrieved from the DOM easily

Reply:
Good to know; will make that change now.